observability/README.md

```diff
@@ -32,6 +32,8 @@ spec:
 
 3. The applications launched with `dapr run` will by default reference the config file in `$HOME/dapr/config.yaml` or `%USERPROFILE%\dapr\config.yaml` and can be overridden with the Dapr CLI using the `--config` param. For example, the following command will launch the hello-world quickstart app using the default config.yaml:
 
+4. Clone this repo using `git clone [-b <dapr_version_tag>] https://github.com/dapr/quickstarts.git` and go to the repo's directory via `cd quickstarts/observability`.
```

Comment:
Preferably have 4th point as the 3rd and the current 3rd point as the 4th. Better continuity

Reply:
Why? There is no use of anything in this repo until this point.